package.json

```diff
@@ -26,16 +26,22 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
+    "@reso/reso-certification-etl": "github:RESOStandards/reso-certification-etl",
+    "ajv": "^8.12.0",
+    "ajv-errors": "^3.0.0",
+    "ajv-formats": "^2.1.1",
     "aws-sdk": "^2.1354.0",
     "axios": "^0.24.0",
     "chalk": "^4.1.2",
     "commander": "^9.4.0",
     "dotenv": "^16.0.1",
     "fastest-levenshtein": "^1.0.16",
     "fs-extra": "^10.0.0",
+    "lodash": "^4.17.21",
```

Comment:
Not a huge fan of lodash in particular. Is there something we absolutely need it for?

Reply:
I was using it for deep cloning part of the schema or else there was a major performance issue with AJV (an interesting GC-related bug where I had to go through the profiler to find the issue). But since we are using node 18 I can just use `structuredClone` instead.